index.js

```diff
@@ -33,29 +33,46 @@ https://github.com/solid/solid
  * @main solid-client
  */
 
+var rdf = require('./lib/util/rdf-parser')
+var acl = require('solid-permissions')
+var auth = require('solid-auth-tls')
+var identity = require('./lib/identity')
+var ns = require('solid-namespace')(rdf)
+var webClient = require('solid-web-client')(rdf)
+
 /**
  * @class Solid
  * @static
  */
 var Solid = {
-  acl: require('./lib/permissions/authorization').acl,
+  acl: acl.acl,
   AppRegistration: require('./lib/solid/app-registration'),
   appRegistry: require('./lib/app-registry'),
-  auth: require('./lib/auth'),
+  auth: auth,
   config: require('./config'),
-  currentUser: require('./lib/auth').currentUser,
-  getProfile: require('./lib/identity').getProfile,
-  getPermissions: require('./lib/permissions').getPermissions,
-  clearPermissions: require('./lib/permissions').clearPermissions,
+  currentUser: auth.currentUser,
```

Comment:
Can you confirm a backwards-incompatible API change here? Looks like calls to `solid.getProfile`, for example, will have to update to `solid.auth.getProfile`? Could you add that to the release notes if so?